src/Raven.Server/Config/Categories/MonitoringConfiguration.cs

```diff
@@ -120,5 +122,129 @@ public sealed class SnmpConfiguration : ConfigurationCategory
             [ConfigurationEntry("Monitoring.Snmp.DisableTimeWindowChecks", ConfigurationEntryScope.ServerWideOnly)]
             public bool DisableTimeWindowChecks { get; set; }
         }
+
+        [ConfigurationCategory(ConfigurationCategoryType.Monitoring)]
+        public sealed class OpenTelemetryConfiguration : ConfigurationCategory
+        {
+            [Description("Indicates if OpenTelemetry is enabled or not. Default: false")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.Enabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool Enabled { get; set; }
+            
+            [Description("Indicates if server-wide OpenTelemetry is enabled or not. Default: true")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.ServerWideEnabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool ServerWideEnabled { get; set; }
+            
+            [Description("Indicates if database-wide OpenTelemetry is enabled or not. Default: true")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.DatabaseWideEnabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool DatabaseWideEnabled { get; set; }
+            
+            [Description("Indicates if AspNetCoreInstrumentation metrics are enabled or not. Default: true")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.Metrics.AspNetCoreInstrumentation", ConfigurationEntryScope.ServerWideOnly)]
+            public bool AspNetCoreInstrumentationMetricsEnabled { get; set; }
+            
+            [Description("Indicates if should use OpenTelementry protocol.")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.OpenTelemetryProtocolEnabled", ConfigurationEntryScope.ServerWideOnly)]
+            public bool OltpExporter { get; set; }
+            
+            [Description("Indicates meters should be exposed to console.")]
+            [DefaultValue(false)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.ConsoleExporter", ConfigurationEntryScope.ServerWideOnly)]
+            public bool ConsoleExporter { get; set; }
+            
+            [Description("Expose server storage instruments.")]
+            [DefaultValue(true)]
+            [ConfigurationEntry("Monitoring.OpenTelemetry.ServerStorage", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
We need to work on configuration key conventions here, this should be ServerStorage.Enabled?